conf/battle/monster.conf

```diff
@@ -297,3 +297,7 @@ achievement_mob_share: no
 // Should slaves teleport back to their master if they get too far during chase? (Note 1)
 // Default (Official): no
 slave_stick_with_master: no
+
+// Absolute minimum respawn time of a monster.
```

Comment:
in milliseconds?

Reply:
Agree that the units should be made clear!